spec/ddoc.dd

```diff
@@ -542,12 +546,23 @@ Identifiers in documentation comments that are function parameters or are
 names that are in scope at the associated declaration are emphasized in
 the output.
 This emphasis can take the form of italics, boldface, a hyperlink, etc.
-How it is emphasized depends on what it is - a function parameter, type,
+How it is emphasized depends on what it is &mdash; a function parameter, type,
```

Comment:
There's a problem somewhere around the `&mdash;`, because of which the Latex build fails the following rather cryptic error:
```

! Misplaced alignment tab character &.
l.24904 ...t is emphasized depends on what it is &
                                                  mdash; a function paramete...

? 
! Emergency stop.
```